src/components/Select.vue

```diff
@@ -191,20 +196,13 @@
     font-size: 1em;
     display: inline-block;
     border: 1px solid transparent;
+    border-left: none;
     outline: none;
     margin: 4px 0 0 0;
-    padding: 0 0.5em;
+    padding: 0 7px;
     max-width: 100%;
     background: none;
     box-shadow: none;
-
-    /* `flex-grow` will stretch the input to take all remaining space, but We
-       need to ensure a small amount of space so there's room to type input. We'll
-       set the input to "hidden" (via width: 0) when the dropdown is closed, to
-       prevent adding a "blank" line (see: https://github.com/sagalbot/vue-select/pull/512).
-       In that case, the flex-grow will still stretch the input to take any
-       available space, on the same "line."
-    */
```

Comment:
@sagalbot I think this, or something like it, is still needed, or we'll get the extra "blank line".

![vue_select_dev](https://user-images.githubusercontent.com/48658/45164384-530d0700-b1c0-11e8-9fed-24a6fb2eeb36.jpg)

It seems that adjusting the `width: 0;` (two lines down) will fix that problem. In the image below I've done that and outlined the inputs in red, so we can see where they are. Seems to work in all cases?

![vue_select_dev](https://user-images.githubusercontent.com/48658/45165156-2b1ea300-b1c2-11e8-8b0d-b9be73b3903a.jpg)

Reply:
I *think* it's overriding the browser's default behavior of growing the input to some minimum width. I've tried setting `min-width: 0` and `flex-basis: 0` instead, but they didn't work. So...as you say, 🤷‍♀️.